code/game/gamemodes/gang/dominator.dm

```diff
@@ -13,6 +13,7 @@
 	var/datum/gang/gang
 	var/operating = 0	//0=standby or broken, 1=takeover
 	var/warned = 0	//if this device has set off the warning at <3 minutes yet
+	var/spam_prevention = 100
```

Comment:
Ok but while I'm here I gotta ask about the definition fetish. For everyone 1 time I've thought "wow that was nice only having to change a single definition" or "Gee this definition helps me understand how this code works" I've had 5 "oh god damnit what number is this definition where did you guys hide it?"